app/controllers/account_reset/recover_controller.rb

```diff
@@ -1,51 +0,0 @@
-module AccountReset
-  class RecoverController < ApplicationController
-    include TwoFactorAuthenticatable
-
-    before_action :confirm_two_factor_enabled
-    before_action :confirm_user_verified
-
-    def show
-      analytics.track_event(Analytics::IAL2_RECOVERY_REQUEST_VISITED)
-    end
-
-    def create
-      analytics.track_event(Analytics::IAL2_RECOVERY_REQUEST, analytics_attributes)
-      Recover::CreateRecoverRequest.call(current_user.id)
-      send_notifications
-      redirect_to account_reset_recover_email_sent_url
-    end
-
-    private
-
-    def send_notifications
-      current_user.confirmed_email_addresses.each do |email_address|
-        UserMailer.confirm_email_and_reverify(
-          current_user,
-          email_address,
-          current_user.account_recovery_request,
-        ).deliver_now
```

Comment:
Should this method and email template be removed?

https://github.com/18F/identity-idp/blob/073bf899d3ea723346a0ffaba5c4fd42687e6f14/app/mailers/user_mailer.rb#L148-L153

https://github.com/18F/identity-idp/blob/stevegsa-rm-ial2-recovery/app/views/user_mailer/confirm_email_and_reverify.html.erb